modules/gltf/src/lib/api/normalize-gltf-v1.ts

```diff
@@ -298,7 +298,7 @@ class GLTFV1Normalizer {
         roughnessFactor: 1
       };
 
-      const textureId = material.values && material.values.tex;
+      const textureId = material.values?.tex || material.values?.texture2d_0;
```

Comment:
Can we add a link to glTF 1 specification describing this? I can't find it. Is this just a random application specific field or a common choice? https://github.com/KhronosGroup/glTF/tree/main/specification/1.0#materials-and-shading

Reply:
`tex` is what we've seen in data before. `texture2d_0` is what we have in this bug. I don't see the texture description in the specification.